### DISCUSSION TITLE
APISIX-Base如何理解？

### DISCUSSION THREAD

**Discussion by liuguangyu133** [apache/apisix#7876] [Q&A]
![image](https://user-images.githubusercontent.com/113016850/188830350-abccba5d-a724-407e-8b5b-61dfa156dec4.png)
gzip插件需要运行在APISIX-Base上，看了下介绍感觉APISIX-Base大概就是需要一些openresty额外的组件所以需要重新自己编译一份，所以我理解如果默认yum安装的应该不支持这个插件才对，但是测试过了一下gzip这个插件是能正常使用并且生效了的，所以请问下是哪个地方理解有问题呢

**Comment by tzssangglass** [comment]
APISIX-Base is the OpenResty distribution of APISIX. If you install the rpm for APISIX via yum, then apisix-base is installed by default
